ui/packages/docs/docs/reference/characters/travelerhydro.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Traveler (Hydro)
+---
+
+import HitlagTable from "@site/src/components/Hitlag/HitlagTable";
+import FieldsTable from "@site/src/components/Fields/FieldsTable";
+import ParamsTable from "@site/src/components/Params/ParamsTable";
+import FramesTable from "@site/src/components/Frames/FramesTable";
+import IssuesTable from "@site/src/components/Issues/IssuesTable";
+import AoETable from "@site/src/components/AoE/AoETable";
+import NamesList from "@site/src/components/Names/NamesList";
+import ActionsTable from "@site/src/components/Actions/ActionsTable";
+
+## Frames
+
+<FramesTable item_key="travelerhydro" />
```

Comment:
Missing frame docs, copy the info from another traveler variant and swap out the element specific sheet/video links.